bundle/compliance/policy/gcp/compute/ensure_no_use_of_default_sa.rego

```diff
@@ -0,0 +1,31 @@
+package compliance.policy.gcp.compute.ensure_default_sa
+
+import data.compliance.policy.gcp.data_adapter
+import future.keywords.every
+import future.keywords.if
+
+is_default_sa(sa) if {
+	endswith(sa.email, "-compute@developer.gserviceaccount.com")
+}
+
+is_gke_instance(instance) if {
+	startswith(instance.name, "gke-")
+}
+
+is_default_sa_with_access(sa) if {
+	is_default_sa(sa)
+	some scope in sa.scopes
+	scope == "https://www.googleapis.com/auth/cloud-platform"
```

Comment:
CIS didn't explicitly mention this scope but I'm quite sure that this is what they meant.
GCP's [implementation](https://github.com/GoogleCloudPlatform/inspec-gcp-cis-benchmark/blob/4790c2f43785e74af719ac26fafc525a00c885f3/controls/4.02-vms.rb#L58C30-L58C30) reference the same scope.